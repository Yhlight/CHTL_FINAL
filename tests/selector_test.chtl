div {
    class: box;
    style {
        width: 100px;
    }
}

p {
    style {
        width: .box.width + 50px; // 150px
    }
}
