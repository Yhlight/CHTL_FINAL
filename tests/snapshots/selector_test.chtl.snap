<div class="box" style="width: 100px; " />
<p style="width: 150px; " />
